build/configs/imxrt1050-evk/loadable_elf_apps/defconfig

```diff
@@ -296,7 +293,16 @@ CONFIG_IMXRT_HYPERFLASH=y
 #
 # Board-Partition Options
 #
-# CONFIG_ARCH_USE_FLASH is not set
+CONFIG_ARCH_USE_FLASH=y
+
+#
+# Board-Partition Options
+#
+CONFIG_FLASH_PARTITION=y
+CONFIG_FLASH_MINOR=0
+CONFIG_FLASH_PART_SIZE="256,256,256,256,4096,"
+CONFIG_FLASH_PART_TYPE="none,none,ftl,ftl,smartfs,"
```

Comment:
Smartfs Partition is created but is not enabled in this defconfig. Please enable smartfs.

Reply:
It is not related to this commit. will modify later.